### PR TITLE
Cargo gets 150 credits when someone opens their mail

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Gives a mutually beneficial relationship so cargo techs are incentivized to get people their mail

To my knowledge there is no incentives like this besides being a good Samaritan and doing your job

It's not much but it's an ok amount

# Changelog

:cl: 
tweak: Cargo is now rewarded with credits when someone opens their mail
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
150 is still pretty large honestly considering how much mail there is and how trivial the action is. I'd like to see it be 100 instead.

**Comment by ynot01** [comment]
> 150 is still pretty large honestly considering how much mail there is and how trivial the action is. I'd like to see it be 100 instead.

Let's say you make a round trip and 10 people open their mail, which is generous; Cargo made 1500 credits off of that work, which I think is completely fair.

File: code/game/objects/items/mail.dm [hunk 1]
```diff
@@ -113,13 +113,15 @@
...
 /obj/item/mail/attack_self(mob/user)
+	var/recipient_real = FALSE // Whether there is a recipient
 	if(recipient_ref)
 		var/datum/mind/recipient = recipient_ref.resolve()
 		// If the recipient's mind has gone, then anyone can open their mail
 		// whether a mind can actually be qdel'd is an exercise for the reader
 		if(recipient && recipient != user?.mind)
 			to_chat(user, span_notice("You can't open somebody else's mail! That's <em>illegal</em>!"))
 			return
+		recipient_real = !!recipient // This will be true if recipient resolved successfully
```
Comment (Bibby0110) [inline code/game/objects/items/mail.dm]:
```suggestion
		recipient_real = recipient // This will be true if recipient resolved successfully
```

**Comment by UltraBigRat** [comment]
> 150 is still pretty large honestly considering how much mail there is and how trivial the action is. I'd like to see it be 100 instead.

its pretty fair considering how ridicilous amounts of money are already gained by cargo for just selling ores miners get in ludicrous amounts. 150 is basically nothing

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**